pyproject.toml

```diff
@@ -3,3 +3,9 @@ profile = "black"
 
 [tool.djlint]
 ignore="H021"
+
```

Comment:
Not sure if this is the direction we want to go with this, will create a card to revisit this if we do implement the filtering.

Reply:
I don't think we should be doing this. After some reading it seems there is a migration path to use something else, with that in mind I think we need to explore that first, rather than just ignoring in the code. I am not against that being put on a separate story as it could be chunky.

Reply:
Will look to remove and split out this part of the PR into a separate card.